### DISCUSSION TITLE
강좌 리스트

### DISCUSSION THREAD

**Discussion by parkmina365** [InChil2/SlotTagging#8] [참고자료]
[딥러닝을 이용한 자연어 처리](https://www.edwith.org/ai331/joinLectures/195055?isDesc=false)

**Comment by InChil2** [comment]
[자연어 처리 기법 실무활용 방법](https://www.youtube.com/watch?v=OOfCI8R0jr8)

**Comment by InChil2** [comment]
[ BERT로 Q&A 구현해보기 With SQuAD AND KERAS](https://www.youtube.com/watch?v=LuApA264Wbs)

**Comment by InChil2** [comment]
[Colab에서 케라스 BERT로 네이버 영화 감성분석 따라하기](https://www.youtube.com/watch?v=OOfCI8R0jr8)

**Comment by sumunoh** [comment]
[자연어 언어모델 'BERT' 1~5강 링크](https://youtu.be/qlxrXX5uBoU)

**Comment by parkmina365** [comment]
[딥러닝 기반 자연어 언어모델 BERT_T아카데미.pdf](https://github.com/InChil2/SlotTagging/files/7386310/BERT.pdf)

**Comment by InChil2** [comment]
[Flask 웹 구현 강좌](https://docs.microsoft.com/ko-kr/learn/modules/python-flask-build-ai-web-app/)
